### ISSUE TITLE
Plugin for Android studio 3.5

### ISSUE THREAD

**Issue by ispbox** [JetBrains/spek#773]
AS 3.5 has just released. Do you have any estimations to update Spek plugin to support it?

I found version 2.0.6 here (https://plugins.jetbrains.com/plugin/10915-spek-framework/versions), and it should be supported by AS 3.5, but my AS 3.5 says it is not supported (I downloaded it manually).
Automatic installation of Spek Framework plugin also doesn't help.

Android Studio 3.5
Build #AI-191.8026.42.35.5791312, built on August 9, 2019
JRE: 1.8.0_202-release-1483-b03 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by raniejade** [comment]
Probably just need to change the version range in the build file. Thanks for the report!

**Comment by raniejade** [comment]
@ispbox can you try this build out? 
[2.1.0-alpha.0.11+20190827T081910Z-Studio3.5.zip](https://github.com/spekframework/spek/files/3545025/2.1.0-alpha.0.11%2B20190827T081910Z-Studio3.5.zip)

**Comment by ispbox** [comment]
@raniejade I just tried, and it solves the problem only partially.
Now all Spek tests in project don't have error icons, and Android Studio doesn't suggest to configure the plugin.
But, unfortunately, it is not possible to execute tests - "Run" button gets disabled if I select any Spek test.
I tried to open source file with Spek tests and run from there. Run button is enabled, and even popup menu is shown, but nothing happens on click.

**Comment by raniejade** [comment]
It's not so easy after all :cry:, I'll find some time this weekend to take a look.

**Comment by ispbox** [comment]
I guess the change is due to new mechanism of running simultaneously on several devices.
Thanks a lot for your efforts, because it is difficult to live without tests.

**Comment by raniejade** [comment]
@ispbox This is working fine for me (using the sample android project). Do note that I have freshly installed AS3.5.

**Comment by ispbox** [comment]
@raniejade Sorry for misleading you... I reinstalled AS and I was able to run my tests. 

But later I've found the cause of my problems: my Spek tests are located in different modules, and in order to run them simultaneously in AS 3.4 (and before) I had an empty test, where I added all other Spek tests in configuration block "Before launch". So it is convenient - I run only one test, and AS automatically launches all nested tests from all modules.

And it looks like that this feature is causing my problems. I'm able to run any single Spek test, but once I run my top-level test, everything gets to the state I've described earlier: nothing is running. After AS restart I'm once again able to run single tests, but also once again top-level test breaks everything.

I tried to delete test configuration and recreated it manually from the beginning. But it didn't help.

So thank you: now I'm able to run single tests, but I hope you will find a fix for the case above. Thanks in advance!
